### PR TITLE
Sentry install

### PR DESCRIPTION
**Pull request by darmenise**
Added Sentry for production exception notifications. See: https://github.com/getsentry/sentry-ruby

### CODE REVIEW

**Review by kanchan-bluebash — COMMENTED** [review]
We can merge this.

**Review by kanchan-bluebash — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -31,6 +31,9 @@ gem 'recaptcha'
 # PDF generation
 gem 'wicked_pdf'
 gem 'wkhtmltopdf-binary-edge'
+# Error notification
+gem 'sentry-rails'
+gem 'sentry-ruby'
```
Comment (darmenise) [inline Gemfile:36]: Wouldn't it be better to move these into the `group :production` only? If so, we also need to update the initializer below.

File: config/initializers/sentry.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  # Set tracesSampleRate to 1.0 to capture 100%
+  # of transactions for performance monitoring.
+  # We recommend adjusting this value in production
+  config.traces_sample_rate = 0.5
+  # or
+  # config.traces_sampler = lambda do |context|
+  #   true
+  # end
+end
```
Comment (darmenise) [inline config/initializers/sentry.rb:13]: If we move the gems into the :production group, then we need to embed embet this into a `if defined?` block, something like: 

```
if defined? Sentry
  Sentry.init do |config|
   config.dsn = ENV['SENTRY_DNS']
   [...]
   end
end
```

**Review by darmenise — COMMENTED** [review]
@vishal-bluebash what do you think about this?

**Pull request merged**